docs/datagrid/columns.md

```diff
@@ -95,14 +95,16 @@ events: {
 ## Sorting
 The DataGrid allows you to set the querying options. The basic one is sorting.
 
-You have three states of sorting:
- * 1 - disabled
- * 2 - descending
- * 3 - ascending
+Sorting object consists of two main configuration options:
+- **`sortable`**: A boolean that determines whether a column can be sorted by the user clicking on the column header.
 
-When the sortable option is turned on, clicking on the header triggers the next state in order to the current one.
+- **`order`**: Specifies the initial sorting order for a column. It can be set to `'asc'` (ascending), `'desc'` (descending), or `null`, which indicates that initial sorting is disabled for that column. Note that only one column can have an `order` set at a time; if multiple columns are specified, only the last one will be considered.
```

Comment:
```suggestion
- **`order`**: Specifies the initial sorting order for a column. It can be set to `'asc'` (ascending) or `'desc'` (descending). If `order` is defined in multiple columns only the last one will be considered.
```